subscription_oca/tests/test_subscription_oca.py

```diff
@@ -0,0 +1,211 @@
+# Copyright 2023 ooops404
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import fields
+from odoo.tests import TransactionCase
+
+
+class TestSubscriptionOCA(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.pricelist = cls.env["product.pricelist"].create(
+            {"name": "pricelist for contract test"}
+        )
+        cls.partner = cls.env["res.partner"].create(
+            {
+                "name": "partner test subscription_oca",
+                "property_product_pricelist": cls.pricelist.id,
+                "email": "demo1@demo.com",
+            }
+        )
+        cls.partner_2 = cls.env["res.partner"].create(
+            {
+                "name": "partner test subscription_oca 2",
+                "property_product_pricelist": cls.pricelist.id,
+                "email": "demo2@demo.com",
+            }
+        )
+        cls.tax_10pc_incl = cls.env["account.tax"].create(
+            {
+                "name": "10% Tax incl",
+                "amount_type": "percent",
+                "amount": 10,
+                "price_include": True,
+            }
+        )
+        cls.product_1 = cls.env.ref("product.product_product_1")
+        cls.product_1.taxes_id = [(6, 0, cls.tax_10pc_incl.ids)]
+        cls.product_2 = cls.env.ref("product.product_product_2")
+        cls.country_id = cls.env["res.country"].search([], limit=1)
```

Comment:
```suggestion
        cls.country = cls.env["res.country"].search([], limit=1)
```
to do for all vars that are records and not IDs